Add possibility to use default fiori styles in all renderers (#36)
kyma-incubator/documentation-component: packages/odata-render-engine/README.md

## Installation

- using [`npm`](https://www.npmjs.com/)

  ```bash
  npm i @kyma-project/dc-odata-render-engine
  ```

- using [`yarn`](https://yarnpkg.com/en/)

  ```bash
  yarn add @kyma-project/dc-odata-render-engine
  ```

## Prerequisites

- [`odata-react`](https://github.com/kyma-incubator/documentation-component/tree/master/packages/odata-react): ^0.2.0-alpha.2
- [`styled-components`](https://github.com/styled-components/styled-components): >= 3.4.0

### Styles

> To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/), install [this](https://sap.github.io/fundamental/getting-started.html) npm package and import the styles as follows:

Reviewer: ```suggestion
To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/), install the npm package from [this](https://sap.github.io/fundamental/getting-started.html) Getting Started guide and import the styles as follows:
```

> To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/) follow its [Getting Started](https://sap.github.io/fundamental/getting-started.html) guide and import it like follows:

Reviewer: ```suggestion
To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/), install [this](https://sap.github.io/fundamental/getting-started.html) npm package and import the styles as follows:
```
